pom.xml

```diff
@@ -45,7 +45,18 @@
           <configuration>
             <excludeFilterFile>src/findbugs/excludesFilter.xml</excludeFilterFile>
             <failOnError>${maven.findbugs.failure.strict}</failOnError>
+            <xmlOutput>true</xmlOutput>
+            <findbugsXmlWithMessages>true</findbugsXmlWithMessages>
```

Comment:
See https://jenkins.ci.cloudbees.com/job/plugins/job/git-plugin/872/findbugsResult/ for example